docs/setup/start-stop.asciidoc

```diff
@@ -46,4 +46,10 @@ include::install/init-systemd.asciidoc[]
 include::install/rpm-init.asciidoc[]
 
 [float]
-include::install/systemd.asciidoc[]
\ No newline at end of file
+include::install/systemd.asciidoc[]
+
+=== Homebrew packages
```

Comment:
add [float] above this line, so it looks like this:

[float]
=== Homebrew packages